.github/workflows/e2e-tests.yml

```diff
@@ -1,9 +1,21 @@
-name: Release tests
+name: E2E Tests
 
-on: push
+on:
+  # Only run this on pushes to master
+  push:
+    branches:
+    - master
```

Comment:
is this late feedback, waiting to run this until pushes on master? I'm not sure in our workflows when we want this to run, but it's worth giving some thought to.

is the purpose of this
- for daily master runs to get early warning if the latest versions of suite components will fail e2e tests?
- for feedback on PRs where we bump the suite release version to know if something is broken before we tag a new suite release?
- other?

Reply:
I see below it (if I'm reading correctly) also runs on PRs - but it's still worth asking how we want this to run / what it's purpose is

Reply:
Since there is no other way to get code on master other than PRs and force-pushes, this change makes sure that our main branch is protected from failures and that we don't run these tests all the time needlessly.

These e2e tests will run:
- anytime master is updated (they should always be green if we do the git workflow)
- anytime a PR is created or modified (which should prevent merges of code to master that's failing the e2e tests)